README.md

```diff
@@ -132,73 +133,63 @@ import java.sql.*;
  * Scenario 1: Failover happens when autocommit is set to true - Catch SQLException with code 08S02.
  */
 public class FailoverSampleApp1 {
-  private static final String CONNECTION_STRING = "jdbc:mysql:aws://database-mysql.cluster-XYZ.us-east-2.rds.amazonaws.com:3306/myDb";
-  private static final String USERNAME = "username";
-  private static final String PASSWORD = "password";
-  private static final int MAX_RETRIES = 5;
-
-  public static void main(String[] args) throws SQLException {
-    // Create a connection.
-    try(Connection conn = DriverManager.getConnection(CONNECTION_STRING, USERNAME, PASSWORD)) {
-      // Configure the connection.
-      setInitialSessionState(conn);
-   
-      // Do something with method "betterExecuteQuery" using the Cluster-Aware Driver.
-      String select_sql = "SELECT * FROM employees";
-      try(ResultSet rs = betterExecuteQuery(conn, select_sql)) {
-        while (rs.next()) {
-          System.out.println(rs.getString("name"));
-        }
+   private static final String CONNECTION_STRING = "jdbc:mysql:aws://database-mysql.cluster-XYZ.us-east-2.rds.amazonaws.com:3306/employees";
```

Comment:
Oh no 😱 , I might have copy pasted the code with wrong spacing. Can you please double check the spacing?